.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+    - 6
+    - 6.0
```

Comment:
because `6` refers always to the latest release of node 6 and I thought it could be important to know, if we might break things with older `6.xx` versions.

Just because for example that `Buffer.allocUnsafe` would build fine with `4` but not with `4.0`